html_css/v2/main-course/transforms-animation.md

```diff
@@ -0,0 +1,176 @@
+### Introduction
+
+Ready to add some animated flair to your website? CSS transitions and animations can add that "panache" to your elements so your websites will set you apart from the rest of the boring crowd. Animate your buttons, nav-bars, loading-bars, anything! Your imagination is your limit! Now let's dive in.  
+
+### Learning Outcomes
+
```

Comment:
learning outcomes should be statements not questions.

something like:

- You'll learn 2 methods to create animations, and when to use them.
- You'll learn how to make sure your animations are smooth and performant.

If you think these questions are important move them to 'knowledge check'